app/views/hyrax/dashboard/collections/_form.html.erb

```diff
@@ -19,9 +19,9 @@
       <% end %>
     <% end %>
   </ul>
-  <%= simple_form_for @form, url: [hyrax, :dashboard, @form], html: { class: 'editor' } do |f| %>
 
-  <div class="tab-content">
+  <%= simple_form_for @form, url: [hyrax, :dashboard, @form], html: { class: 'editor' } do |f| %>
```

Comment:
Can you say more about the changes made to this file?

Reply:
The tags were not aligned.  I added comments to make it clearer the tag closing an open.  I cleaned up the indentations to make things easier to read.  This helped me to verify that there was indeed a missing close `</div>`.  I added the missing close tag.